### PR TITLE
Convert latest_reading_recorded_at value to UTC, and save the raw value

### PR DESCRIPTION
**Pull request by equivalentideas**
This PR:

1. Adds a new column to capture the raw `lastest_reading_recorded_at` value, and populates it with the existing values (`latest_reading_recorded_at_raw`)
2. Extracts the processing and serialization of the scraped values to a new `Aqm::Reading` Class.
3. Changes the existing `latest_reading_recorded_at` to UTC time, and changes the behaviour so future recordings are in this format too.

It felt right to name space `Reading` under a `Aqm` module. Maybe we should do the same with `AqmRecord` ? I don't mind at this point.

Part of https://github.com/equivalentideas/westconnex_M4_East_Air_Quality_Monitoring/issues/32

### CODE REVIEW

File: lib/reading.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      self.no2_concentration = hash['no2_concentration']
+      self.differential_temperature_lower = hash['differential_temperature_lower']
+      self.differential_temperature_upper = hash['differential_temperature_upper']
+      self.wind_speed = hash['wind_speed']
+      self.wind_direction = hash['wind_direction']
+      self.sigma = hash['sigma']
+    end
+
+    def latest_reading_recorded_at_converted
+      return unless latest_reading_recorded_at_raw && !latest_reading_recorded_at_raw.empty?
```
[the remark this answers is not in the document]
Comment (henare) [reply lib/reading.rb]: Ha - I [stand corrected](https://rubocop.readthedocs.io/en/latest/cops_style/#styleguardclause)!

**Comment by henare** [comment]
@equivalentideas ah damn, looks like I can't request a review from you even though I now want you to review my changes above :smile:

**Comment by equivalentideas** [comment]
looks great @henare very nice pracrice around keep the migration self contained,i hadn't thought about that before.

There's a typo in test description '+10 an incorrectly labeld', should be 'and'.

Go ahead and ship this if you like. I'm going to be afk this weekend.

**Comment by henare** [comment]
@equivalentideas thanks for checking this over. Getting the go-ahead to merge and deploy was just what I needed :construction: 

This is now deployed :rocket: Post-deployment checks suggest this is all working as expected. Backup `b033` was captured prior to this deployment.

**Pull request merged**